Xcode 8 and ENABLE_BITCODE error when building using Carthage

Can anyone help with this?
- Xcode 8
- Carthage

When I do an update it falls over when building Dobby. Other projects are fine.

**\* Building scheme "Dobby-iOS" in Dobby.xcworkspace
*\* BUILD FAILED **

The following build commands failed:
    Ld /Users/scott/Library/Developer/Xcode/DerivedData/Dobby-ecuwfbyesepmhtfviqolgxllubry/Build/Intermediates/Dobby.build/Release-iphoneos/Dobby-iOS.build/Objects-normal/arm64/Dobby normal arm64
    Ld /Users/scott/Library/Developer/Xcode/DerivedData/Dobby-ecuwfbyesepmhtfviqolgxllubry/Build/Intermediates/Dobby.build/Release-iphoneos/Dobby-iOS.build/Objects-normal/armv7/Dobby normal armv7

ld: warning: Auto-Linking supplied '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest.tbd', bitcode bundle could not be generated because '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest.tbd' was built without full bitcode. All frameworks and dylibs for bitcode must be generated from Xcode Archive or Install build
ld: warning: Auto-Linking supplied '/Applications/Xcode.app/Contents/Developer/Toolchains/Swift_2.3.xctoolchain/usr/lib/swift/iphoneos/libswiftXCTest.dylib', '/Applications/Xcode.app/Contents/Developer/Toolchains/Swift_2.3.xctoolchain/usr/lib/swift/iphoneos/libswiftXCTest.dylib' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target.
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: warning: Auto-Linking supplied '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest.tbd', bitcode bundle could not be generated because '/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest.tbd' was built without full bitcode. All frameworks and dylibs for bitcode must be generated from Xcode Archive or Install build
ld: warning: Auto-Linking supplied '/Applications/Xcode.app/Contents/Developer/Toolchains/Swift_2.3.xctoolchain/usr/lib/swift/iphoneos/libswiftXCTest.dylib', '/Applications/Xcode.app/Contents/Developer/Toolchains/Swift_2.3.xctoolchain/usr/lib/swift/iphoneos/libswiftXCTest.dylib' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target.
clang: error: linker command failed with exit code 1 (use -v to see invocation)
A shell task (/usr/bin/xcrun xcodebuild -workspace /Volumes/Mobi/Mobi2GoInStoreManager/Carthage/Checkouts/Dobby/Dobby.xcworkspace -scheme Dobby-iOS -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
*\* BUILD FAILED **

--- a/Dobby.xcodeproj/project.pbxproj
+++ b/Dobby.xcodeproj/project.pbxproj
@@ -587,6 +587,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphonesimulator*]" = "";
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
+				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PLATFORM_DIR)/Developer/Library/Frameworks",
@@ -605,6 +606,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphonesimulator*]" = "";
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
+				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PLATFORM_DIR)/Developer/Library/Frameworks",

